images/dotnet/metadata.yaml

```diff
@@ -4,7 +4,18 @@ logo: https://storage.googleapis.com/chainguard-academy/logos/dotnet.svg
 endoflife: ""
 console_summary: ""
 short_description: Minimal image for .NET and the .NET Tools.
-compatibility_notes: ""
+compatibility_notes: |
+    The Chainguard .NET images are available on `cgr.dev` as two variants: `dotnet-sdk` and `dotnet-runtime`. The SDK variant contains additional tooling to facilitate development and building, while the runtime variant contains only the runtime to execute .NET applications. Both the `sdk` and `runtime` images also have `latest-dev` version that contain a shell and various other tools for development.
```

Comment:
Ah now that you've made this change, run `monopod readme` and it will render the main `README.md` file with this included.

Reply:
So when I attempt to do this, I am met with the following error (executed in the root directory of the repo)
```
Error: <nil>: Configuration file not found; The configuration file images/apko/metadata.hcl does not exist.
2024/03/13 11:25:44 error during command execution: <nil>: Configuration file not found; The configuration file images/apko/metadata.hcl does not exist.
```
And when I run `monopod readme render --image dotnet` I get the following error.
```
Error: <nil>: Configuration file not found; The configuration file images/dotnet/metadata.hcl does not exist.
2024/03/13 11:28:04 error during command execution: <nil>: Configuration file not found; The configuration file images/dotnet/metadata.hcl does not exist.
```
Is there a flag or command that I am missing when I execute this?